k8s/go-echo-deploy.yml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Unless there was a specific reason for choosing this `k8s/` directory for the deploy manifests, I would recommend we use `deploy/` for consistency with other repos like KIC. If you wanted to convert this to `kustomize` manifests, could even put it in `config/` as that's common. In any case, I would say try to structure it similar to [KIC](https://github.com/kong/kubernetes-ingress-controller).